exasol/nb_connector/text_ai_extension_wrapper.py

```diff
@@ -186,63 +231,81 @@ def initialize_text_ai_extension(
             If True allows overriding the language definition.
     """
 
-    # Create the name of the Exasol connection object
-    db_user = str(conf.get(CKey.db_user))
-    bfs_conn_name = "_".join([BFS_CONNECTION_PREFIX, db_user])
-    # As soon as the official release of TXAIE is available, the hard-coded value for
-    # container_name can be replaced by TXAIELanguageContainerDeployer.SLC_NAME,
-    # see https://github.com/exasol/notebook-connector/issues/179.
-    container_name = "exasol_text_ai_extension_container_release.tar.gz"
+    def install_slc(version=None, container_file=None):
+        container_url = version and TXAIELanguageContainerDeployer.slc_download_url(
+            version
+        )
+        deploy_language_container(
+            conf,
+            container_name=TXAIELanguageContainerDeployer.SLC_NAME,
+            container_file=container_file,
+            container_url=container_url,
+            language_alias=language_alias,
+            activation_key=ACTIVATION_KEY,
+            path_in_bucket=PATH_IN_BUCKET,
+            allow_override=allow_override,
+        )
 
-    def from_ai_lab_config(key: CKey) -> Path | None:
-        entry = conf.get(key)
-        return Path(entry) if entry else None
+    # Create the name of the Exasol connection to the BucketFS
+    db_user = conf.get(CKey.db_user)
+    bfs_conn_name = f"{BFS_CONNECTION_PREFIX}_{db_user}"
 
     if run_deploy_container:
+        print("Text AI: Downloading and installing Script Language Container (SLC)")
         if version:
-            install_text_ai_extension(version)
+            install_slc(version)
             # Can run_upload_models, run_deploy_scripts,
             # run_encapsulate_bfs_credentials, etc. be ignored here?
             return
```

Comment:
the return needs to go, I think, I think, if we check first for container_file, can use then

`version = verson or importlib...`